recipes/pdal/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "pdal" %}
+{% set version = "1.7.1" %}
+{% set sha256 = "b780b64e68bf68cf31530a7b5822228303c671516c8282a6c6e19c3f27fbb345" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://download.osgeo.org/{{ name }}/PDAL-{{ version }}-src.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win or not py36]
+
+requirements:
+  build:
+    - cmake
+    - toolchain
+    - numpy 1.11.*
+    - gdal 2.2.4
```

Comment:
Think the current recommendation for this one is `gdal 2.2.*`. @ocefpaf would know more than I though.